cmd/ci-operator-prowgen/main.go

```diff
@@ -175,18 +188,15 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to process arguments")
 	}
 
-	args := flagSet.Args()
```

Comment:
I'm not sure we want to remove this functionality in this PR - sure, it might be useful in the future to remove it, but by not having it in this specific PR you don't block yourself on migrating users, etc